src/components/static/pages/AboutUs.tsx

```diff
@@ -2,116 +2,257 @@ import * as React from "react";
+import IzzyReal from "../../../assets/images/profile/izzy-irl.png";
+import Aaron from "../../../assets/images/profile/aaron-thumbnail.png";
+import AaronReal from "../../../assets/images/profile/aaron-profile-irl.png";
+import Dee from "../../../assets/images/profile/dee-thumbnail.png";
+import DeeReal from "../../../assets/images/profile/dee-profile-irl.png";
 
-const AboutUsWrapper = styled.div`
-  background-color: #d2d2d2;
-  height: ${contentHeight}px;
-`;
+import headerimg from "../../../assets/images/our-team/team.png";
+import backgroundImageWithNuvi from "../../../assets/images/2018_0814_Pattern_Adjusted.svg";
 
-const QuoteWrapper = styled.div`
-  background-color: #d2d2d2;
-  height: 430px;
-`;
+import {
+    Col,
+    Grid,
+    Row
+} from "react-bootstrap";
 
-const VisionWrapper = styled.div`
-  background-color: #ffffff;
-  height: 370px;
-`;
 
-const VisualWrapper = styled.div`
-  background-color: #d2d2d2;
-  height: 370px;
+const Header = styled.div`
+    background-image: url(${headerimg});
+    width: 100%;
+    height: 100%;
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position:center;
+    background-size:cover;
+    box-shadow:0 1px 0 black,0 2px 0 rgba(255,255,255,0.15);
+    padding: 300px;
+    font-size: 100px;
+    text-align: center;
+    color: white;
+
 `;
-const MissionWrapper = styled.div`
-  background-color: #ffffff;
-  height: 370px;
+
+const HeaderText = styled.div`
+       
+    text-align: center;
+    color: white;
+    font-size: 100px;
+    opacity: 0;
+    animation: fade-in-right ease 2s forwards;
+`
+
+const Background = styled.div`
+  background-repeat: none;
+  background-image: url(${backgroundImageWithNuvi});
+  font-family: "Lato", sans-serif;
+  
 `;
-const MessageWrapper = styled.div`
-  background-color: #d2d2d2;
-  height: 370px;
+
+const Bio = styled.div`
+    font-family: "Lato", sans-serif;
+    width: 700px;
+    margin: 100px;
+    padding: 50px;
+    background: #d0e6e8;
+    font-size: 18px;
 `;
-const OurTeamWrapper = styled.div`
-  background-color: #ffffff;
-  height: 370px;
+
+const Bio1 = styled(Bio)`
+    background: #fce68f
+
 `;
-const BoardWrapper = styled.div`
-  background-color: #ffffff;
-  height: 370px;
+
+const BioPic = styled.div`
+    position: relative;
+    display: inline-block;
+    margin:100px;
+    float: center;
 `;
 
+class TeamMember {
+    name: string;
+    role: string;
+    quote: string;
+    bio: string;
+    img: string;
+    cartoon: string;
+
+    constructor(name: string, role: string, quote: string, bio: string, img: string, cartoon: string) {
+        this.name = name;
+        this.role = role;
+        this.quote = quote;
+        this.bio = bio;
+        this.img = img;
+        this.cartoon = cartoon;
+    }
+}
+
+
 export class AboutUs extends React.Component {
-  public render(): any {
-    return (
-      <AboutUsWrapper>
-        <QuoteWrapper>
-          <div className="quote-text">
```

Comment:
Since we've removed most of the old code in this component, we should remove the unused css classes inside `AboutUs.css`.